packages/common/dlq.py

```
@@ -0,0 +1,165 @@
+        Args:
+            redis_client: Redis client for queue operations.
+            max_retries: Maximum retry attempts (default: 3).
+            base_delay_seconds: Base delay for exponential backoff (default: 2).
+        """
+        self.redis_client = redis_client
+        self.max_retries = max_retries
+        self.base_delay_seconds = base_delay_seconds
+
+        logger.info(
+            f"Initialized DeadLetterQueue (max_retries={max_retries}, "
+            f"base_delay={base_delay_seconds}s)"
```

Ruff (0.14.1)

Logging statement uses f-string

(G004)